### PR TITLE
Mojo Helpdesk - new components

### LINKED ISSUE #3691: Mojo Helpdesk
### App
Mojo Helpdesk

### Triggers
* New Ticket
* New Ticket with Due On
* New Ticket with Scheduled On
* New User

### Actions
* Create Ticket
* Create User

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #3691

### CODE REVIEW

File: components/mojo_helpdesk/actions/create-user/create-user.mjs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    },
+  },
+  async run({ $ }) {
+    const data = {
+      email: this.email,
+      first_name: this.firstName,
+      last_name: this.lastName,
+      send_welcome_email: this.sendWelcomeEmail
+        ? 1
+        : 0,
```
Comment (luancazarine) [inline components/mojo_helpdesk/actions/create-user/create-user.mjs:42]: You can use the [unary + operator](https://developer.mozilla.org/en/JavaScript/Reference/Operators/Arithmetic_Operators#.2B_%28Unary_Plus%29), which converts its operand into a number.
```suggestion
      send_welcome_email: +this.sendWelcomeEmail,
```

**Review by luancazarine — APPROVED** [review]
Hi @michelle0927. Thanks for your contribution! I've just left a comment, but LGTM!

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Mojo_Helpdesk_1322.pdf](https://github.com/PipedreamHQ/pipedream/files/10964712/Mojo_Helpdesk_1322.pdf)

**Pull request merged**